software/index.md

```diff
@@ -3,19 +3,156 @@ title: Software Development
 has_children: true
 nav_order: 100
 ---
-
 # {{ page.title }}
 
-Development of CESSDA tools and services is carried out using CESSDA-owned git-repositories on GitHub.
+The CESSDA development infrastructure offers a number of benefits for
+potential suppliers of software artefacts for the CESSDA Research
+Infrastructure, not least of which is a ready made development
+environment that can be used at no cost, and will increase the
+likelihood of software artefacts meeting the agreed acceptance criteria.
+
+## Overview
+
+Development of CESSDA tools and services is carried out using CESSDA-owned Git repositories hosted on GitHub.
 
-If the code is developed publicly elsewhere, mirroring with clear pointers to the upstream are used.
+If the code is developed publicly elsewhere, the source code is mirrored with clear pointers present in the repository description to the upstream source.
 
-All software development should follow common best practices and the [DRY-principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
-At a minimum, the recommendations of the
+All software development should follow common best practices for the tools and languages used. Additionally, the recommendations of the
 [EURISE Network Technical Reference, {{ site.technical-reference.version }}]({{ site.technical-reference.url }})
 must be followed.
 
-To assess the fitness for CESSDA purposes, software systems must pass the [quality gate]({% link software/quality-gate.md %}).
+To assess the fitness for CESSDA purposes, software systems must be validated by static analysis tools. CESSDA uses SonarQube as its main static analysis tool, and code must the [quality gate]({% link software/quality-gate.md %}) in order for deployment to proceed.
```

Comment:
Missing a word ".. code must pass the [quality ..."?

Reply:
Good spot, thanks